src/nodes/QskBoxNode.cpp

```diff
@@ -117,7 +117,7 @@ void QskBoxNode::setBoxData( const QRectF& rect,
     {
         if ( isFillMonochrome && isBorderMonochrome )
         {
-            if ( borderColors.color( Qsk::Left ) == fillGradient.startColor() )
+            if ( borderColors.gradient( Qsk::Left ) == fillGradient.startColor() )
```

Comment:
We have to compare the startColors only - not the gradients itself.
Otherwise we might fail with the orientation of the gradient !